src/components/Investigaciones/autorxs/autorxs.json

```diff
@@ -0,0 +1,109 @@
+[
```

Comment:
nitpick: please move all JSONs to a central place so it's easier to track them when we go removing them.

Reply:
@juanigaray is it possible to have TS map these things to a model ?